### ISSUE TITLE
Transferring maintenance to the perl6 project

### ISSUE THREAD

**Issue by treyharris** [hinrik/perl6-mode#20]
Hi @hinrik —

For some time on IRC and over at the [perl6/perl6-mode](perl6/perl6-mode) fork, we've been discussing how to get an updated version of the Emacs perl6-mode onto MELPA.

I'm not sure whether you are interested in continuing to contribute to the package or not, but since there's been no activity from you on this repo for over 9 months, and no responses to issues or PRs, we assume you no longer have the time to devote to it.

We propose the [Perl 6 Project](/perl6) take over maintenance. This proposal is being tracked in perl6/perl6-mode#11. We will request GitHub break the fork between this project and that one, and then will update the MELPA recipe to point to it.

**This should require no intervention on your part** unless you object. Please do not delete your repo until we have completed the task list, so that we can ensure continuity of maintenance and merge any unresolved issues from both repos.

I notice you are not currently a member of the Perl 6 project; we'll be happy to add you if you wish to continue contribution to perl6-mode maintenance or to contribute to other perl6 repos.

If you have any objections to this plan or any other questions or suggestions, please let us know (either here or at perl6/perl6-mode#11) **before Sunday 14 May 2017,** at which point we will proceed with the task list.

Thank you very much for your authorship of this package; a solid Emacs major mode is important to any language's real-world success, and you provided this at a crucial juncture. We all very much appreciate your work, and your name will always be first among the authors. But to continue driving Perl 6's adoption among Emacs users even "after Christmas", we'll need to keep the package up-to-date with new language and editor features, and we think increased community involvement will help us to do that.

**Comment by hinrik** [comment]
Yeah, let's transfer it to the perl6 account. I'm sure it'll receive more love there. :)
